packages/core/src/odata/common/constructable.ts

```diff
@@ -1,24 +0,0 @@
-/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
-
-import { EntityBase, EntityBuilderType } from './entity';
-import { RequestBuilder } from './request-builder/request-builder';
-import { CustomFieldBase, Selectable, Field, Link } from './selectable';
-
-/**
- * @hidden
- */
-export interface Constructable<
```

Comment:
I assume this interface is moved to entity.ts without changes.

Reply:
yes just moved to the entity.